src/components/Ratings/Ratings.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import { RadioButton as CoreRadio } from 'wix-ui-core/radio-button';
+import styles from './Ratings.st.css';
+import { ReactComponent as StarIcon } from '../../assets/icons/Star.svg';
+import { RATINGS_DATA_HOOKS, RATINGS_DATA_KEYS } from './dataHooks';
+import classNames from 'classnames';
+
+export enum Size {
+  Small = 'small',
+  Large = 'large',
+}
+
+export enum Mode {
+  Input = 'input',
+  Display = 'display',
+}
+
+export enum Layout {
+  Aside = 'aside',
+  Bottom = 'bottom',
+}
+
+export interface RatingsProps {
+  value: number;
+  mode: Mode;
+  onSelect?(value: number): void;
+  inputOptions?: [string, string, string, string, string];
+  disabled?: boolean;
+  error?: boolean;
+  size?: Size;
+  layout?: Layout;
+  ratingDisplay?: string;
+  countDisplay?: string;
+  'data-hook'?: string;
+}
+
+interface DefaultProps {
+  value: number;
+  disabled: boolean;
+  error: boolean;
+  mode: Mode;
+  size: Size;
+  layout: Layout;
+}
+
+/** Ratings component based on IconToggle */
+export class Ratings extends React.Component<RatingsProps> {
+  static displayName = 'Ratings';
+  static defaultProps: DefaultProps = {
+    value: 0,
+    disabled: false,
+    error: false,
+    mode: Mode.Input,
+    size: Size.Small,
+    layout: Layout.Aside,
+  };
+
+  getDataAttributes() {
+    const { disabled, error, size } = this.props;
+
+    return {
+      [RATINGS_DATA_KEYS.Disabled]: disabled,
+      [RATINGS_DATA_KEYS.Error]: error,
+      [RATINGS_DATA_KEYS.Size]: size,
+    };
+  }
+
+  _renderContent = () => <StarIcon />;
+
+  _renderInputOptions = () => {
+    const { inputOptions, value } = this.props;
+
+    return (
+      <div className={styles.labelList}>
+        {inputOptions.map((el, idx) => (
```

Comment:
The problem is that `inputOptions` doesn't have a default value, so this might explode.
Maybe we should have `inputOptions` to be an empty array as a default.